packages/nitro-protocol/test/contracts/examples/SingleAssetPayments/validTransition.test.ts

```diff
@@ -40,7 +40,7 @@ describe('validTransition', () => {
     ${true}  | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${4}     | ${{A: 1, B: 0, C: 2}} | ${'B pays C 1 wei'}
     ${true}  | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${5}     | ${{A: 1, B: 2, C: 0}} | ${'C pays B 1 wei'}
     ${false} | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${5}     | ${{A: 0, B: 2, C: 1}} | ${'A pays B 1 wei (not their move)'}
-    ${false} | ${[1, 1]} | ${[false, false]} | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 1, B: 2, C: 1}} | ${'Guarantee'}
+    ${false} | ${[1, 1]} | ${[false, false]} | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 0, B: 2, C: 1}} | ${'Guarantee'}
```

Comment:
This test case currently passes for the wrong reason. The transition is invalid because the total amount allocated changed. That's not the desired failure mode to probe, here. This change makes the test fail by making the transition valid according to the current contract implementation.

Reply:
Meta comment: this test would be improved if it asserted on the revert message. That would have caught this problem earlier.